cli/command/load_balancer_test.go

```diff
@@ -0,0 +1,40 @@
+package command
+
+import (
+	"testing"
+
+	"github.com/quintilesims/layer0/common/models"
+	"github.com/urfave/cli"
+)
+
+func TestCreateLoadBalancer(t *testing.T) {
+	base, ctrl := newTestCommand(t)
+	defer ctrl.Finish()
+
+	loadBalancerCommand := NewLoadBalancerCommand(base.Command())
+	c := &cli.Context{}
+	loadBalancerCommand.create(c)
+
+	ports := []models.Port{}
```

Comment:
You typically shouldn't use 0-values when testing things of this nature. It's easier to catch bugs when things aren't 0-values Ports should have at least one object, healthcheck should be populated, isPublic should be true.